index.html

```diff
@@ -22,6 +16,12 @@
 etherpad:         # optional (insert the URL for your Etherpad if you're using one)
 eventbrite:       # optional (insert the alphanumeric key for Eventbrite registration, e.g., "1234567890AB")
 ---
+
+
+<!-- See instructions in the comments below for how to edit specific sections of this workshop template. -->
```

Comment:
This comment probably belongs as a separate paragraph in the HEADER comment immediately below it.